File: 03-CPP/04-condicionales/04switch.cpp
// Switch en C++
#include <iostream>
using namespace std;

int main()
{

    int diaSemana;
    cout << "Ingresa un dia de la semana (1 - 3)" << endl;
    cin >> diaSemana;

    switch (diaSemana)
    {
    case 1:
        printf("Es lunes");
        break;
    case 2:
        printf("Es martes");
        break;
    case 3:
        printf("Es miercoles");
        break;
    default:
        cout << "Valor invalido" << endl;
    }

    return 0;
}

File: 04_CSHARP/04_Condicionales/04switch.cs
// Switch en C#

int diaSemana;
Console.WriteLine("Ingresa un dia de la semana 1 - 3");
diaSemana = Convert.ToInt32(Console.ReadLine());

switch (diaSemana)
{
    case 1:
        Console.WriteLine("Lunes");
        break;
    case 2:
        Console.WriteLine("Martes");
        break;
    case 3:
        Console.WriteLine("Miercoles");
        break;
    default:
        Console.WriteLine("Valor no valido");
}